main.py

```diff
@@ -2,23 +2,25 @@
 import sys
 
 from scrapper.nomad_list_scrapper import NomadListScrapper
-from conf import JSON_FILENAME
+from conf import JSON_FILENAME, NOMAD_LIST_URL
 from logger import Logger
+from scrapper.web_driver import WebDriver
+
 
 def main():
     logger = Logger().logger
+    web_driver = WebDriver(logger, NOMAD_LIST_URL)
     try:
-        cities = NomadListScrapper(logger).get_cities()
+        nomad_scrapper = NomadListScrapper(logger, web_driver)
```

Comment:
Much more testable I think (because we are injecting the web_driver now). Well done!